Article/Article.js

```diff
@@ -3,24 +3,30 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
+    // this.content = document.querySelector('.article p');  **don't need**
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
```

Comment:
Could also be written as `this.expandButton.addEventListener('click', this.expandArticle.bind(this))` just a matter of preference though.